library/src/iosMain/kotlin/com/lightningkite/kiteui/views/direct/ScrollLayout.kt

```diff
@@ -60,6 +60,8 @@ class ScrollLayout : UIScrollView(CGRectZero.readValue()), UIViewWithSizeOverrid
 
         if (viewDebugTarget?.native === this) println("Total sizeThatFits $subsize")
 
+        subsize.primary = subsize.primary.coerceAtMost(mySizeWithoutPadding.primary)
```

Comment:
very suspicious

Reply:
This is correct. Clarifying comment added